src/ru/crazycoder/plugins/tabdir/TitleFormatter.java

```diff
@@ -109,6 +105,9 @@ private static String removeDuplicates(final String key, Set<String> neighbours)
 
     public static String format(List<String> prefixes, String tabName, FolderConfiguration configuration) {
         String joinedPrefixes = joinPrefixes(prefixes, configuration);
+        if (joinedPrefixes.isBlank()) {
+            return configuration.getEmptyPathReplacement() + MessageFormat.format(configuration.getTitleFormat(), joinedPrefixes, tabName);
```

Comment:
MessageFormat.format calculated in both cases, can be extracted to variable

Reply:
ok, moved it to a variable in a new commit